tutorials/polkadot-sdk/system-chains/asset-hub/send-a-tx-paying-fee-different-token.md

```diff
@@ -0,0 +1,167 @@
+---
+title: Send a Transaction While Paying the Fee with a Different Token
+description: This tutorial demonstrates how to send a DOT transfer transaction while paying the fees using a different token on the Asset Hub.
+---
+
+# Send a Transaction While Paying the Fee with a Different Token
```

Comment:
What about something like this? Pay Transaction Fees in a Different Token

It just feels a bit clunky as it is now